app/controllers/admin/settings/campaigns_controller.rb

```diff
@@ -1,6 +1,11 @@
 module Admin
   module Settings
     class CampaignsController < Admin::Settings::BaseController
+      private
+
+      def authorization_resource
```

Comment:
Was there a problem without this? In general, if the entity we deal with matches the controller name, the authorization resource will be correctly inferred, so we only need to add it for controllers where the controller name doesn't match.